plugins/labhub.py

```diff
@@ -96,7 +96,7 @@ def TEAMS(self, new):
         self._teams = new
 
     # Ignore LineLengthBear, PycodestyleBear
-    @re_botcmd(pattern=r'(?:(?:invite)|(?:inv))\s+(?:(?:@?([\w-]+)(?:\s*(?:to)\s+(\w+))?)|(me))',
+    @re_botcmd(pattern=r'^(?:(?:invite)|(?:inv))\s+(?:(?:@?([\w-]+)(?:\s*(?:to)\s+(\w+))?)|(me))',
                re_cmd_name_help='invite [to team]')
```

Comment:
I tested it in gitter chat also, its working fine

Reply:
That is insufficient, as regressions can occur if there are no tests.